### PR TITLE
Jshimkus rh patch 1

### PR DESCRIPTION
**Pull request by jshimkus-rh**
what the what

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0-only
```
Comment (jshimkus-rh) [inline Makefile:1]: test body

Comment (jshimkus-rh) [inline Makefile:1]: test body

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: test body

Comment (jshimkus-rh) [inline Makefile:1]: test body

Comment (jshimkus-rh) [inline Makefile:1]: test body

Comment (jshimkus-rh) [inline Makefile:1]: test body

Comment (jshimkus-rh) [inline Makefile:1]: test body

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This is comment at the file level 4/12/23, 10:47

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: test body

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: test body

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This is comment at the file level 4/12/23, 10:47

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This is comment at the file level 4/12/23, 10:47

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: 'This is comment at the file level 4/12/23, 10:47'

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This is comment at the file level 4/12/23, 10:47

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: 'This is comment at the file level 4/12/23, 10:47'

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: 'This is comment at the file level 4/12/23, 10:47'

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: 'This is comment at the file level 4/12/23, 10:47'

File: src/tools/README.txt [hunk 2]
```diff
@@ -41,3 +41,5 @@ upgrader:	Automated upgrade mechanism.
 virt:		Source to our "permabit-virt" Debian package: for 
 		manipulating a virtual network interface, used in
 		some automated tests.
+
+jbs: some additional text just to create a PR.
```
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: Line 1 comment

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: Line 1 comment

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: What the what?

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: 'April 12, 2023 at 14:09:47 EDT'

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: 'line 45'

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: 'line 44 to 45'

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: doo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: edlkjadeflkj

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: ff

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: fopp

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: tewt

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: fpop

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

File: src/tools/README.txt [hunk 3]
```diff
@@ -41,3 +41,5 @@ upgrader:	Automated upgrade mechanism.
 virt:		Source to our "permabit-virt" Debian package: for 
 		manipulating a virtual network interface, used in
 		some automated tests.
+
```
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:44]: f

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

File: src/tools/README.txt (hunk 3 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: fpp

Comment (jshimkus-rh) [inline src/tools/README.txt:44]: f

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: this is lines fourty-four and forty-five.

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: This is a post-change file comment.

File: Makefile [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 #
-# Copyright Red Hat
```
Comment (jshimkus-rh) [inline Makefile:3]: should keep 'Red Hat'

File: Makefile [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
+# Copyright The Great Bird of The Galaxy
```
Comment (jshimkus-rh) [inline Makefile:3]: should not have changed

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This is a post-change README comment.

Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This is a pre-change README comment.

File: src/tools/README.txt (hunk 3 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: this is a blank line

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: this is a fantastic change

**Comment by jshimkus-rh** [comment]
what the what

**Comment by jshimkus-rh** [comment]
pr comment

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: new Makefile file comment

Comment (jshimkus-rh) [inline Makefile:1]: old Makefile file comment

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: new line one comment

File: Makefile (hunk 4 above)
Comment (jshimkus-rh) [inline Makefile:3]: old line one comment

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: new README comment

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: multi-line

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: comment from generalized REST api.

File: src/tools/README.txt (hunk 3 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: line

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: This day is call'd the feast of Crispian. He that outlives this day, and comes safe home, Will stand a tip-toe when this day is nam'd, And rouse him at the name of Crispian. He that shall live this day, and see old age, Will yearly on the vigil feast his neighbours, And say 'To-morrow is Saint Crispian.' Then will he strip his sleeve and show his scars, And say 'These wounds I had on Crispian's day.' Old men forget; yet all shall be forgot, But he'll remember, with advantages, What feats he did that day. Then shall our names, Familiar in his mouth as household words- Harry the King, Bedford and Exeter, Warwick and Talbot, Salisbury and Gloucester- Be in their flowing cups freshly rememb'red. This story shall the good man teach his son; And Crispin Crispian shall ne'er go by, From this day to the ending of the world, But we in it shall be remembered- We few, we happy few, we band of brothers; For he to-day that sheds his blood with me Shall be my brother; be he ne'er so vile, This day shall gentle his condition; And gentlemen in England now-a-bed Shall think themselves accurs'd they were not here, And hold their manhoods cheap whiles any speaks That fought with us upon Saint Crispin's day.

**Comment by jshimkus-rh** [comment]
fileCommentInfo

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: print("Processing comments", end = "")

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: print("Processing comments", end = "")

**Comment by jshimkus-rh** [comment]
print("Processing comments", end = "")

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: print("Processing comments", end = "")

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: print("Processing comments", end = "")

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: comment

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: copy

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: comment

**Comment by jshimkus-rh** [comment]
Overall commment April 14, 2023 at 18:40:09 EDT

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: File comment April 14, 2023 at 18:40:29 EDT

File: Makefile (hunk 4 above)
Comment (jshimkus-rh) [inline Makefile:3]: left line comment

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: right line comment

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: File comment April 14, 2023 at 18:41:35 EDT

File: src/tools/README.txt (hunk 3 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: single-line comment

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: multi-line comment

**Comment by jshimkus-rh** [comment]
general comment

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: readme

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: new

**Comment by jshimkus-rh** [comment]
commn

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: file

**Comment by jshimkus-rh** [comment]
pull

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: file

File: src/tools/README.txt (hunk 3 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: a44

**Comment by jshimkus-rh** [comment]
pull

File: src/tools/README.txt
Comment (jshimkus-rh) [inline src/tools/README.txt:1]: file

File: src/tools/README.txt (hunk 3 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: a44

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: 45

**Comment by jshimkus-rh** [comment]
PULL LEVEL - April 21, 2023 at 09:53:48 EDT

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: PULL LEVEL - April 21, 2023 at 09:53:48 EDT

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: PULL LEVEL - April 21, 2023 at 09:53:48 EDT

**Comment by jshimkus-rh** [comment]
pull comment

**Comment by jshimkus-rh** [comment]
Probably shows up as jshimkus-rh

File: Makefile
Comment (jshimkus) [inline Makefile:1]: How 'bout this?

File: Makefile (hunk 5 above)
Comment (jshimkus) [inline Makefile:3]: and this?

**Review by jshimkus — CHANGES_REQUESTED** [review]
Previous didn't seem to take.

File: Makefile (hunk 1 above)
Comment (jshimkus) [inline Makefile:1]: Review comment on line?  There was only one so maybe/maybe not.

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: 'line 3 from jshimkus'

**Comment by jshimkus** [comment]
globqal

**Review by jshimkus-rh — COMMENTED** [review]
This is the review body.

**Review by jshimkus — COMMENTED** [review]
This should be from jshimkus (using that token).

**Review by jshimkus — COMMENTED** [review]
?

**Comment by jshimkus-rh** [comment]
fff

File: src/tools/README.txt (hunk 3 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:44]: dddd

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: argh!!!!

Comment (jshimkus) [inline src/tools/README.txt:45]: single comment

Comment (jshimkus) [inline src/tools/README.txt:45]: aaa

**Review by jshimkus — COMMENTED** [review]
finishing aaa

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus-rh) [inline src/tools/README.txt:45]: foo

**Review by jshimkus — COMMENTED** [review]
starting a review to comment

File: src/tools/README.txt (hunk 2 above)
Comment (jshimkus) [inline src/tools/README.txt:45]: ffff

**Review by jshimkus — COMMENTED** [review]
finihing

**Comment by jshimkus-rh** [comment]
General comment

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: lien 3

**Comment by jshimkus-rh** [comment]
general

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: file

File: Makefile (hunk 5 above)
Comment (jshimkus) [inline Makefile:3]: a line

**Review by jshimkus — COMMENTED** [review]
the review

File: Makefile (hunk 4 above)
Comment (jshimkus) [inline Makefile:3]: left

File: Makefile (hunk 5 above)
Comment (jshimkus) [inline Makefile:3]: right

**Review by jshimkus — COMMENTED** [review]
done

**Comment by jshimkus-rh** [comment]
general

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: makefile

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: line

File: Makefile
Comment (jshimkus-rh) [inline Makefile:1]: file

File: Makefile (hunk 5 above)
Comment (jshimkus-rh) [inline Makefile:3]: line

**Pull request closed without merging**